1.css

```diff
@@ -0,0 +1,298 @@
+
+@import url('https://fonts.googleapis.com/css?family=Montserrat');
+
+* {
+    margin: 0px;
+    padding: 0px;
+}
+body {
+    height: 100%;
+  }
+
+p{
+    display: inline-block;
+    font-family: 'Montserrat', sans-serif;
+    color:white;
+    font-size: 1.5em;
+    padding-left: 5%;
+    padding-top: 40px;
+    padding-bottom: 40px;
+    /* font-weight:bold; */
+}
+p:nth-child(1){
+padding-left: 54px;
+}
+
+header{
+    background-color:#373c3f;
+    width: 100%;
+    height: 100%;
+}
+
+div.banner{
+    height: 48%;
+    padding: 0%;
+    background-color: #0ca2b8;
+    
+}
+div.media{
+    height: 38%;
+    padding: 0%;
+    overflow: hidden;
+    width: 100%;
+} 
+.banner>p {
+    padding-top:5%;
+    padding-bottom:5%;
+    font-weight:bold; 
+    font-size: 3em;
+    padding-left: 15%;
+    display: block;
+   
+}
+.banner :nth-child(2) {
+    margin: 0px;
+    padding-top: 0%;
+    font-size: 2em;
+    opacity: 0.5;
+    display: inline-block;
+}
+.banner :nth-child(3) {
+    margin: 0px;
+    padding-top: 0%;
+    font-size: 50px;
+    padding-left: 15%;
+    display: inline-block;
+}
+.textbox{
+width: 100%;
+overflow: hidden;
+background-color: #27212c;
+}
+.left{
+    float: left;
+    height: 100%;
+    width: 80%;
+}
+.left>p{
+color: #9282a0;
+
+font-family: 'Montserrat', sans-serif;
+padding-top: 80px;
+padding-left: 20%;
+padding-bottom: 40px;
+font-size: 30px;
+}
+.right{
+    float: right;
+    height: 100%;
+    width: 20%;
+    padding-bottom: 0%;
+    background-color: #bc3183;
+}
+.right>p{
+    padding-left: 40%;
+    padding-top: 80px;
+    padding-bottom: 40px;
+font-size: 50px;
+font-weight: 700;
+line-height: 65px;
+}
+
+
+.media img{
+float: left;
+    width: 33.333%;
+    height: 100%;
+    max-height: 600px;
+}
+.Info{
+  padding-top: 7%; 
+  overflow: hidden;
+  height: 100%;
+  background-color: #ebebeb;
+}
+
+.Info h2 {
+    padding-left: 7.5%;
+    padding-bottom: 2%;
+    font-size: 60px;
+    font-weight: lighter;
+    color: #439cb0;
+}
+
+.Info > div > article {
+    font-size: 1.2em;
+    padding: 3.5%;
+    display: inline-block;
+}
+.Info > div > article {
+    overflow-wrap: break-word;
+    font-family: 'Montserrat', sans-serif;
+}
+.secondParagraph {
```

Comment:
Avoid camel case. You could try to use [Yandex BEM](https://en.bem.info/methodology/) for example